feature-libs/product-configurator/rulebased/core/state/effects/configurator-place-order-hook.effect.ts

```diff
@@ -17,40 +18,41 @@ import { ConfiguratorActions } from '../actions/index';
  */
 export class ConfiguratorPlaceOrderHookEffects {
   @Effect()
-  placeOrder$: Observable<ConfiguratorActions.RemoveConfiguration> = this.actions$.pipe(
-    ofType(CheckoutActions.PLACE_ORDER),
-    map(() => {
-      const ownerKeys = [];
-      this.activeCartService
-        .getEntries()
-        .pipe(take(1))
-        .subscribe((entries) => {
-          entries.forEach((entry) => {
-            if (
-              !entry.product?.configurable ||
-              entry.product?.configuratorType !== ConfiguratorType.VARIANT
-            ) {
-              return;
-            }
-            const owner = ConfiguratorModelUtils.createOwner(
-              CommonConfigurator.OwnerType.CART_ENTRY,
-              String(entry.entryNumber)
-            );
+  placeOrder$: Observable<ConfiguratorActions.RemoveConfiguration> = this.eventService
```

Comment:
Does not work. It seems when the event is fired, activeCartService no longer returns the cart entries (which is understandable), so our code that cleans up configurations is not executed.
And, btw: Is it ok to have this class annotated and named as effect? As it does not deal with an action, but with an event?